Makefile

```diff
@@ -1,15 +1,51 @@
-QA_DOCKER_IMAGE=jakzal/phpqa:alpine
+QA_DOCKER_IMAGE=rollerworks/search-phpqa:latest
 QA_DOCKER_COMMAND=docker run -it --rm -v "$(shell pwd):/project" -w /project ${QA_DOCKER_IMAGE}
-ELASTICSEARCH_DOCKER_COMPOSE=lib/Elasticsearch/docker-compose.yml
 
-dist: cs-full phpstan test-full
+dist: install cs-full phpstan test-full
 ci: cs-full-check phpstan test-full
 
+install:
+	composer.phar install --no-progress --no-interaction --no-suggest --optimize-autoloader --ansi
```

Comment:
Can we use composer from Docker or something? Hardcoding `composer.phar` here doesn't seem right